PEP8

Recent Travis build shows PEP8 error
see https://travis-ci.org/scls19fr/APC/builds/337234965

```
0.28s$ flake8 --version
3.5.0 (mccabe: 0.6.1, pycodestyle: 2.3.1, pyflakes: 1.6.0) CPython 2.7.14 on Linux
The command "flake8 --version" exited with 0.
0.76s$ flake8
./apc/lockfile.py:24:1: E722 do not use bare except'
./apc/lockfile.py:64:9: E722 do not use bare except'
./apc/lockfile.py:200:5: E741 ambiguous variable name 'l'
./apc/utility.py:139:13: E722 do not use bare except'
./build/lib/apc/lockfile.py:24:1: E722 do not use bare except'
./build/lib/apc/lockfile.py:64:9: E722 do not use bare except'
./build/lib/apc/lockfile.py:200:5: E741 ambiguous variable name 'l'
./build/lib/apc/utility.py:139:13: E722 do not use bare except'
The command "flake8" exited with 1.
```

--- a/apc/lockfile.py
+++ b/apc/lockfile.py
@@ -21,7 +21,7 @@ try:
     from os import readlink
     from os import remove as rmlink
     _windows = False
-except:
+except Exception:
     _windows = True
 
     try:
@@ -61,7 +61,7 @@ except:
         f.close()
         try:
             rename(newlinkname, filename)
-        except:
+        except Exception:
             os.remove(newvalname)
             os.rmdir(newlinkname)
             raise

--- a/apc/lockfile.py
+++ b/apc/lockfile.py
@@ -197,13 +197,13 @@ def isLocked(name):
     @rtype: C{bool}
     @return: True if the lock is held, False otherwise.
     """
-    l = FilesystemLock(name)
+    lock = FilesystemLock(name)
     result = None
     try:
-        result = l.lock()
+        result = lock.lock()
     finally:
         if result:
-            l.unlock()
+            lock.unlock()
     return not result
 
 

--- a/apc/utility.py
+++ b/apc/utility.py
@@ -136,7 +136,7 @@ class AbstractAPC:
             try:
                 outlet = int(outlet)
                 return (outlet, 'Outlet #%d' % outlet)
-            except:
+            except Exception:
                 raise SystemExit('Bad outlet: [%s]' % outlet)
 
     def _escape_to_main(self, depth=6):

